package.json

```diff
@@ -79,7 +79,7 @@
         "@swc/cli": "^0.1.62",
         "@swc/core": "^1.3.95",
         "@swc/jest": "^0.2.29",
-        "@telefonica/acceptance-testing": "5.0.0",
+        "@telefonica/acceptance-testing": "5.1.0",
```

Comment:
unrelated, just upgrading this lib to have the changes in https://github.com/Telefonica/js-toolbox/pull/107